Simperium/SPPersistentMutableDictionary.h

```diff
@@ -16,13 +16,38 @@
 
 @interface SPPersistentMutableDictionary : NSObject
 
+/// The Dictionary's Label is used to define the Persistent Store identifier. Different labels will map to different persistent databases.
+///
 @property (nonatomic, strong, readonly) NSString *label;
 
+/// Specifies the Supported Types
+/// - Note: All classes specified here must conform to NSSecureCoding
+///
+@property (nonatomic, strong, readwrite) NSSet<Class> *supportedObjectTypes;
+
+/// Indicates if the stored `Supported Object Types` should be required to conform to NSCoding. Defaults to YES
+///
+@property (nonatomic, assign, readwrite) BOOL requiringSecureCoding;
+
```

Comment:
Is `requiringSecureCoding` only used for unit tests? Maybe it's good to mention it here.

Reply:
YES! Only for testing purposes, good call!!